types/relay-runtime/lib/util/isPromise.d.ts

```diff
@@ -0,0 +1 @@
+export function isPromise(p: any): boolean;
```

Comment:
You should make this a [type-guard](https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards) instead:

```suggestion
export function isPromise(p: any): p is Promise<any>;
```

Which allows you to do e.g.:

```ts
const p = Promise.resolve() as unknown;
if (isPromise(p)) {
    p.then(() => console.log("Indeed a promise"))
}
```

Reply:
(And please also add a test case that uses it like in my example.)